components/Snackbar/src/private/MDCSnackbarOverlayView.m

```diff
@@ -542,6 +542,20 @@ - (void)setBottomOffset:(CGFloat)bottomOffset {
 
     self.bottomConstraint.constant = -[self dynamicBottomMargin];
     [self triggerSnackbarLayoutChange];
+
+    // If there is no snackbar the following method returns CGRectNull, but we still need to notify
+    // observers of bottom offset changes.
+    CGRect frame = [self snackbarRectInScreenCoordinates];
+    if (CGRectIsNull(frame)) {
+      frame = CGRectMake(0,
+                         CGRectGetMaxY(self.frame) - self.bottomOffset,
+                         CGRectGetMaxX(self.frame),
```

Comment:
I would expect this to be CGRectGetWidth(self.frame).
If this isn't GetWidth, consider documenting.